bodhi/server/models.py

```diff
@@ -61,6 +61,133 @@
 except ImportError:
     log.warning("Could not import 'rpm'")
```

Comment:
This looks pretty messed up now. This is not what I meant https://github.com/ankit01ojha/bodhi/blob/b416dcbd62073fa3a495dcd9975c096504e92a0b/bodhi/server/models.py#L53-L60